include/sddf/network/lib_sddf_lwip.h

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2022, UNSW
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <microkit.h>
+#include <sddf/network/queue.h>
+#include "lwip/pbuf.h"
+
+/* Default gigabit link speed. */
+#define SDDF_LWIP_LINK_SPEED 1000000000
```

Comment:
This has always been a constant hardcoded in every component that uses LWIP. It is used to initialise the network interface struct, but doesn't seem to effect runtime behaviour much. I think it may be used for LWIP statistics collection, rather than functioning.